k8s/object.bzl

```diff
@@ -63,10 +63,11 @@ def _impl(ctx):
         # As part of this walk, we also collect all of the image's input files
         # to include as runfiles, so they are accessible to be pushed.
         for tag in ctx.attr.images:
+            resolved_tag = ctx.expand_make_variables("something", tag, {})
```

Comment:
/s/something/tag

Reply:
whoops missed that - thangs for the catch :)

Reply:
Done.